scripts/build_docs_site.sh

```diff
@@ -41,6 +41,20 @@ md_to_adoc() {
         sed -i.bak 's/(\.\.\//(/' $tmpfile
     done
 
+    # Because the antora directory structure is flattened, we need to strip directories deeper than the module level.
+    # This sed takes links that look like "../module/deeper_dir/file.html" and transforms them to "../module/file.html".
+    sed -i.bak 's/\(\](\.\.\/[[:alnum:]_-]*\/\)[^.)]*\/\([[:alnum:]_-]*.html\)/\1\2/g' $tmpfile
```

Comment:
Cant comment on this magic. Thank you for writing the comment at least 😆 
![giphy (1)](https://user-images.githubusercontent.com/12886084/77467732-2efe2e80-6e15-11ea-9e0c-77837c16be3c.gif)